test/utils.js

```diff
@@ -0,0 +1,58 @@
+import * as bowser from 'bowser'
+import db from 'caniuse-db/data.json'
+
+// bowser id -> caniuse id
+const browserIdMap = {
+  msie: 'ie',
+  msedge: 'edge',
+  firefox: 'firefox',
+  chrome: 'chrome',
+  safari: 'safari',
+  opera: 'opera',
+  ios: 'ios_saf',
+  android: 'android',
+  blackberry: 'bb',
+  ucbrowser: 'and_uc',
+  samsungBrowser: 'samsung',
+}
+
+function detectBrowserId() {
+  for (const b in browserIdMap) {
+    if (bowser[b]) return browserIdMap[b]
+  }
+  return 'unknown'
+}
+
+export const browserId = detectBrowserId()
+export const browserVersion = parseFloat(bowser.version)
+
+/**
+ * Query the caniuse database for feature support.
+ *
+ * @param {String} feature - see full list https://github.com/Fyrd/caniuse/tree/master/features-json
+ * @returns {{level: 'unknown' | 'none' | 'partial' | 'full', needPrefix: boolean}}
+ */
+export function getSupport(feature) {
```

Comment:
This library you mentioned earlier doesn't work in the browser: https://github.com/Nyalab/caniuse-api and there was no other library around. Had to do it myself ^^